tensorcircuit/translation.py

```diff
@@ -197,6 +197,8 @@ def qir2qiskit(
             qiskit_circ.measure(index, index)
         elif gate_name == "reset":
             qiskit_circ.reset(index)
+        elif gate_name == "barrier":
+            qiskit_circ.barrier(index)
```

Comment:
is barrier guarantee to be applied on only one index? otherwise, ``barrier(*index)`` may required

Reply:
As of now `QuantumCircuit.barrier` accepts list arguments. I may change it to `barrier(*index)` if it helps to avoid confusion.

Reply:
yes, pls. ``*index`` should be more consistent and clearer